signac/contrib/project.py

```diff
@@ -175,8 +164,7 @@ def __init__(self, config=None):
         )
 
     def __str__(self):
-        """Return the project's id."""
-        return str(self.id)
+        return repr(self)
```

Comment:
For a `Job`, we use `job.id` as the result of `str(job)`. That is, the string form is a simple unique identifier. I think the analogous thing here is just the project's root directory? I'm happy with either solution, but I wanted to pose that idea.
```suggestion
        return str(self.root_directory())
```

Reply:
I agree, that's probably better. The repr is a bit uglier and more verbose than is really needed.